pkg/session/bootstrap.go

```diff
@@ -3575,7 +3575,11 @@ func rebuildAllPartitionValueMapAndSorted(s *session) {
 			if pi == nil || pi.Type != model.PartitionTypeList {
 				continue
 			}
-			tbl, ok := is.TableByID(s.currentCtx, t.ID)
+			ctx := s.currentCtx
+			if ctx == nil {
```

Comment:
Can we change the caller to use a non-nil ctx? Because generally I assume the context should always be non-nil.

Reply:
I find this comment https://github.com/pingcap/tidb/blob/9b1fa1cda02d65f46f85b7f3c5b83d2bf96968bf/pkg/session/session.go#L180

Reply:
Maybe we just use context.Background() here? @lance6716

Reply:
yes I think we can delete this comment and set context.Background() as the default value.